### DISCUSSION TITLE
Enable HTTPS/SSL on fabricjs.com

### DISCUSSION THREAD

**Discussion by apjyotirmay** [fabricjs/fabric.js#6927] [General]
Fabricjs.com doesn't have https access, and in my opinion, it should be enabled even if the page isn't dealing with sensitive content

**Comment by asturur** [comment]
well do you know how to do it on the old github pages?

**Reply by apjyotirmay** [reply]
I'm not sure, but I can look into it. How's this thing working? As in, how is this page being served using the domain name? Pretty sure it's possible, just need to understand where to store the SSL certificates

**Reply by asturur** [reply]
on github.io is obviously possible, with the new website works build in:
https://fabricjs.github.io/
I do not have access to the old .com settings, maybe @kangax knows better.

**Reply by apjyotirmay** [reply]
I see. Looks like the new website would resolve this issue. So it's completely upto you guys if you want to invest anytime doing chores on the current one, or focus completely on the new website

**Reply by asturur** [reply]
I'm not sure when the new website will be ready honestly. It requires a bunch of manual work yet.

**Reply by apjyotirmay** [reply]
Yeah, that's definitely a lot of work. Anyhow, we can figure out the https if you guys have access to the domain and if I can get understanding of the serving part
